kani-compiler/src/codegen_cprover_gotoc/codegen/typ.rs

```diff
@@ -506,10 +506,10 @@ impl<'tcx> GotocCtx<'tcx> {
         self.codegen_ty(typ).to_pointer()
     }
 
-    /// A reference to a Struct<dyn T> { .., data: T} is translated to
+    /// A reference to a Struct\<dyn T\> { .., data: T} is translated to
```

Comment:
I think I'd prefer these to be updated to use code blocks instead, e.g.:

```
/// A reference to a `Struct<dyn T> { .., data: T}` is translated to
```

But either way works for me

Reply:
good call. changing.

Reply:
done